src/DDTrace/Integrations/Yii/V2/YiiIntegrationLoader.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace DDTrace\Integrations\Yii\V2;
+
+use DDTrace\Configuration;
+use DDTrace\Contracts\Scope;
+use DDTrace\GlobalTracer;
+use DDTrace\Http\Urls;
+use DDTrace\Integrations\Yii\YiiSandboxedIntegration;
+use DDTrace\Integrations\Integration;
+use DDTrace\SpanData;
+use DDTrace\Tag;
+use DDTrace\Type;
+use yii\helpers\Url;
+
+class YiiIntegrationLoader
+{
+
+    public function load(YiiSandboxedIntegration $integration)
+    {
+        $scope = GlobalTracer::get()->getRootScope();
+        if (!$scope instanceof Scope) {
+            return Integration::NOT_LOADED;
+        }
+        $root = $scope->getSpan();
+        // Overwrite the default web integration
+        $root->setIntegration($integration);
+        $root->setTraceAnalyticsCandidate();
+        $root->overwriteOperationName('yii.request');
+        $service = Configuration::get()->appName(YiiSandboxedIntegration::NAME);
```

Comment:
This is quite risky actually because we can end up with having an empty service if the user did not set any service name, causing traces to be dropped. I would suggest to go into `Configuration::get()->appName()` and set a default to `'php-web-app'`.

Reply:
The parameter to `Configuration::get()->appName` is a default in the case `DD_SERVICE_NAME` (or one of the other deprecated env variables) is not set or is set but empty. So worst case the service is `yii`.